helm/values-uat.yaml

```diff
@@ -29,13 +29,16 @@ microservice-chart:
         PERSONAL_DATA_VAULT_API_HOST: "https://api.uat.tokenizer.pdv.pagopa.it"
         PERSONAL_DATA_VAULT_API_BASE_PATH: "/tokenizer/v1"
         PAGOPA_MAIL_LOGO_URI: "https://dev.checkout.pagopa.it/assets/logos/pagopa-logo.svg"
+        ELASTIC_APM_SERVICE_NAME: pagopa-notification-service
```

Comment:
add here '-blue' prefix in order to distinguish logs from the blue instance the greed one